src/routes/RequestsRoute.js

```diff
@@ -159,6 +159,25 @@ export const buildHoldRecords = (records) => {
   });
 };
 
+export const REQUEST_ERROR_MESSAGE_CODE = {
+  REQUEST_NOT_ALLOWED_FOR_PATRON_TITLE_COMBINATION: 'REQUEST_NOT_ALLOWED_FOR_PATRON_TITLE_COMBINATION',
+};
+
+export const REQUEST_ERROR_MESSAGE_TRANSLATION_KEYS = {
+  [REQUEST_ERROR_MESSAGE_CODE.REQUEST_NOT_ALLOWED_FOR_PATRON_TITLE_COMBINATION]: 'ui-requests.errors.requestNotAllowedForPatronTitleCombination',
+};
+
+export const getRequestErrorMessage = (error, intl) => {
+  const {
+    code = '',
+    message = '',
+  } = error;
+
+  return code && REQUEST_ERROR_MESSAGE_TRANSLATION_KEYS[code]
+    ? intl.formatMessage({ id: REQUEST_ERROR_MESSAGE_TRANSLATION_KEYS[code] })
```

Comment:
Why do you need an object for just 1 translation key? Why don't you just render `{ id: ui-requests.errors.requestNotAllowedForPatronTitleCombination }`?

Reply:
Hello @mariia-aloshyna Thank you. We will add new transaction key in next stories.